src/components/table/table-cell/table-cell.component.js

```diff
@@ -19,14 +20,41 @@ import OptionsHelper from '../../../utils/helpers/options-helper';
  * set styling options for the cell used for action such as delete.
  */
 class TableCell extends React.Component {
+  /**
+   * Returns if child element has type property.
+   */
+  childHasType(child) {
```

Comment:
This name suggests to me a boolean return value, but actually returns the type if one exists.

Wonder if it might read more clearly if it was called `childType`, and returned `(child && child.type) || undefined`?

Reply:
In fact, looking at where it's used, it might be better to return the `child.type.name` when it exists, as we don't seem to be using it for anything else?